Misc/NEWS.d/next/Tools-Demos/2018-03-16-17-25-05.bpo-29673.m8QtaW.rst

```diff
@@ -0,0 +1,6 @@
+PyUnicode_AsString() was removed in Python and PyUnicode_AsUTF8() is now used.
```

Comment:
Thanks for condensing the NEWS entry.  But I think it is still too detailed and does not really answer the question of *what* was fixed; *why* it was fixed is not so important here.  Also, if you wish, you can add your name here.  An example from the [Developer's Guide](https://devguide.python.org/committing/#what-s-new-and-news-entries):

> Fix warning message when ``os.chdir()`` fails inside
> ``test.support.temp_cwd()``.  Patch by Chris Jerdonek.